<h1 align='center'>Introduction</h1>

![picture](https://www.lovibond.com/ix_pim_assets/Industrien_und_Applikationen/Applikationen/Trinkwasseraufbereitung/MB4/app_drinking_water_mb4_04.jpg)

<p>&emsp;Access to safe drinking-water is essential to health, a basic human right and a component of effective policy for health protection. This is important as a health and development issue at a national, regional and local level. In some regions, it has been shown that investments in water supply and sanitation can yield a net economic benefit, since the reductions in adverse health effects and health care costs outweigh the costs of undertaking the interventions.</p>

<p>&emsp;This notebook will explore features related to water potability and predict water potability using statistics and Machine Learning.</p>

<h1 align='center'>Packages</h1>

In [2]:
import matplotlib.pyplot as plt
import numpy as np
import pandas as pd
import seaborn as sns

from sklearn.linear_model import LogisticRegression
from sklearn.neighbors import KNeighborsClassifier
from sklearn.tree import DecisionTreeClassifier

<h1 align='center'>Data Loading</h1>

In [3]:
df = pd.read_csv("dataset/water_potability.csv")

In [4]:
df.head()

ph    Hardness        Solids  ...  Trihalomethanes  Turbidity  Potability
0       NaN  204.890455  20791.318981  ...        86.990970   2.963135           0
1  3.716080  129.422921  18630.057858  ...        56.329076   4.500656           0
2  8.099124  224.236259  19909.541732  ...        66.420093   3.055934           0
3  8.316766  214.373394  22018.417441  ...       100.341674   4.628771           0
4  9.092223  181.101509  17978.986339  ...        31.997993   4.075075           0

[5 rows x 10 columns]

In [5]:
df.shape

(3276, 10)

There are 3276 rows and 10 columns in the dataset.